rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -0,0 +1,44 @@
+use livesplit_core::{Segment, TimeSpan, TimingMethod};
+use crate::threads::ue::rebo::STATE;
+
+pub struct LiveSplit {
+    pub timer: livesplit_core::Timer,
```

Comment:
I'd prefer `use livesplit_core::Timer as LiveSplitTimer` over specifying the full path everywhere.